automation/src/main/java/org/wso2/testgrid/automation/executor/JMeterExecutor.java

```diff
@@ -120,6 +122,18 @@ public void execute(String script, DeploymentCreationResult deploymentCreationRe
 
         //Persist test scenario
         try {
+            if (testScenario.getTestCases().size() == 0) {
+                testScenario.setStatus(Status.ERROR);
```

Comment:
This is a DID_NOT_RUN?

Reply:
At this point, the scenario has already entered the RUNNING state and JMeter execution has started. So IMO, we should not set it as DID_NOT_RUN. DID_NOT_RUN should be used for scenarios which have not run at all (e.g. due to failure in pipeline) and that will be handled by the the ResolveStatuses command in the post block. WDYT?